staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -374,10 +374,13 @@ func (o *ApplyOptions) Run() error {
 	var objs []runtime.Object
 
 	count := 0
-	err = r.Visit(func(info *resource.Info, err error) error {
-		if err != nil {
-			return err
-		}
+
+	infos, err := r.Infos()
+	if err != nil {
+		return err
+	}
+
+	for _, info := range infos {
```

Comment:
you can reduce the size of this change by wrapping the logic in an local function -- e.g.

```go
for ... {
  err := func() error { ... }
}
```